server/cmd/mmctl/commands/compliance_export.go

```diff
@@ -158,3 +200,85 @@ func complianceExportDownloadCmdF(c client.Client, command *cobra.Command, args
 	printer.Print(fmt.Sprintf("Compliance export file downloaded to %q", path))
 	return nil
 }
+
+func complianceExportCreateCmdF(c client.Client, command *cobra.Command, args []string) error {
+	exportType := args[0]
+	if exportType != model.ComplianceExportTypeActiance &&
+		exportType != model.ComplianceExportTypeCsv &&
+		exportType != model.ComplianceExportTypeGlobalrelay {
+		return fmt.Errorf("invalid export type, must be one of: csv, actiance, globalrelay")
+	}
+
+	dateStr, err := command.Flags().GetString("date")
+	if err != nil {
+		return err
+	}
+	start, err := command.Flags().GetInt("start")
+	if err != nil {
+		return err
+	}
+	end, err := command.Flags().GetInt("end")
+	if err != nil {
+		return err
+	}
+	startTimestamp, endTimestamp, err := getStartAndEnd(dateStr, start, end)
+	if err != nil {
+		return err
+	}
+	startTime := strconv.FormatInt(startTimestamp, 10)
+	endTime := strconv.FormatInt(endTimestamp, 10)
+	exportDir := path.Join(model.ComplianceExportPath, fmt.Sprintf("%s-%s-%s", time.Now().Format(model.ComplianceExportDirectoryFormat), startTime, endTime))
+
+	// If start and end are 0, we need to not set those keys in the job data.
+	// This will make the job like a manual job (it will pick up where the previous job left off).
+	data := model.StringMap{
+		shared.JobDataInitiatedBy:  "mmctl",
+		shared.JobDataExportType:   exportType,
+		shared.JobDataBatchStartId: "",
+		shared.JobDataJobStartId:   "",
+	}
+	if startTimestamp != 0 && endTimestamp != 0 {
+		data[shared.JobDataBatchStartTime] = startTime
+		data[shared.JobDataJobStartTime] = startTime
+		data[shared.JobDataJobEndTime] = endTime
+		data[shared.JobDataExportDir] = exportDir
+	}
+
+	job := &model.Job{
+		Type: model.JobTypeMessageExport,
+		Data: data,
+	}
+
+	if job, _, err = c.CreateJob(context.TODO(), job); err != nil {
+		return fmt.Errorf("failed to create compliance export job: %w", err)
+	}
+
+	printer.Print(fmt.Sprintf("Compliance export job created with ID: %s", job.Id))
+
+	return nil
+}
+
+// getStartAndEnd returns the start and end timestamps in unix milliseconds
+func getStartAndEnd(dateStr string, start int, end int) (int64, int64, error) {
+	if dateStr == "" && start == 0 && end == 0 {
+		// return 0 so that the job will be like a manual job
+		return 0, 0, nil
+	}
+
+	if dateStr != "" && (start > 0 || end > 0) {
+		return 0, 0, errors.New("if date is used, start and end must not be set")
+	}
+
+	if dateStr != "" {
+		t, err := time.Parse("2006-01-02 -0700", dateStr)
+		if err != nil {
+			return 0, 0, fmt.Errorf("could not parse date, use the format with time zone offset: YYYY-MM-DD -0700, eg for EDT: `2024-12-24 -0400`,  error details: %w", err)
```

Comment:
Let's pass the `dateStr` in the error message as well. And `eg for` is not grammatically correct. `e.g.` means `for example`. So we can just keep `e.g. EDT:`.